### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by avarabyeu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,209 @@
...
+As Cucumber runs your features, it calls out to any number of listener objects to let them know 
+how it’s progressing. These listeners are notified at various points throughout the run of features. 
+This principle is used to notify ReportPortal about your tests progress in real-time. 
+ReportPortal supports two kinds of Reporters. 
+Both of them allow you to report your execution progress to ReportPortal, 
+but there are some differences in report structure.  
+
+* **Scenario Reporter** 
+* **Step Reporter**
+Step Reporter propagetes the most traditional for ReportPortal test structure
```
Comment (ksuglotova) [inline README.md]: propagAtes

File: README.md [hunk 2]
```diff
@@ -1,7 +1,209 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+**Parameters**
+
+User should provide next parameters to agent.
+
+| **Parameter**                                 | **Description**      | **Required**|
+|-----------------------------------------------|----------------------|-------------|
+|rp.enable                                      |Enable/Disable logging to report portal: rp.enable=true - enable log to RP server.  Any other value means 'false': rp.enable=false - disable log to RP server.  If parameter absent in properties file then automation project results will be posted on RP. |No |
```
Comment (ksuglotova) [inline README.md]: If parameter is skipped

File: README.md [hunk 3]
```diff
@@ -1,7 +1,209 @@
...
+|rp.username                                    |User name |Yes |
+|rp.password                                    |User password. **We strongly recommend to use UUID** or separate ReportPortal internal users password here to avoid domain password publishing. |Yes |
+|rp.uuid                                        |UUID of user. |Yes |
+|rp.endpoint                                    |URL of web service, where requests should be send |Yes |
```
Comment (ksuglotova) [inline README.md]: should be sent

File: README.md [hunk 4]
```diff
@@ -1,7 +1,209 @@
...
+|rp.launch                                      |The unique name of Launch (Run), based on that name a history of runs will be created for particular name |Yes |
```
Comment (ksuglotova) [inline README.md]: for the particular name

File: README.md (hunk 1 above)
Comment (avarabyeu) [inline README.md]: fixed

File: README.md [hunk 5]
```diff
@@ -1,7 +1,209 @@
...
+|rp.project                                     |Project name, to identify scope |Yes |
+|rp.tags                                        |Set of tags for specifying additional meta information for current launch. Format: tag1;tag2;build:12345-6. Tags should be separated by “;”. There are one special tag- build – it should be used for specification number of build for launch. |No |
+|rp.batch.size.logs                             |In order to rise up performance and reduce number of requests to server |Yes |
+|rp.keystore.resource                           |Put your JKS file into resources and specify path to it | |
+|rp.keystore.password                           |Access password for JKS (certificate storage) package, mentioned above | |
+|rp.convertimage                                |Colored log images can be converted to grayscale for reducing image size. Values: ‘true’ – will be converted. Any other value mean false. |No |
```
Comment (ksuglotova) [inline README.md]: value means

File: README.md [hunk 6]
```diff
@@ -1,7 +1,209 @@
...
+
+**Parameters loading order**
+
+Client loads properties in the next order. Every next level overrides previous:
+
+a.  Properties file. Client loads all known to him properties (specified in the
+    "Input Parameters" section) from "reportportal.properties" file.
+
+b.  Environment variables. If environment variables with names specified in the
+    "Input Parameters" section exists client reload existing properties from
```
Comment (ksuglotova) [inline README.md]: variables exist

File: README.md [hunk 7]
```diff
@@ -1,7 +1,209 @@
...
+    environment variables.
+
+c.  JVM variables. If JVM variables with names specified in the
+    "Input Parameters" section exists, client overrides existing ones from
```
Comment (ksuglotova) [inline README.md]: variables exist

File: README.md [hunk 8]
```diff
@@ -1,7 +1,209 @@
...
+    JVM variables.
+
+#### Events
+
+* Before - used to finish previously started hood
```
Comment (ksuglotova) [inline README.md]: hooK

File: README.md [hunk 9]
```diff
@@ -1,7 +1,209 @@
...
+| CLASS                |Not in structure. Avoid it     |
+| BEFORE_METHOD        |TestItem (type = BEFORE_METHOD)|
+| METHOD               |TestItem (type = STEP)         |
+| AFTER_METHOD         |TestItem (type = AFTER_METHOD) |
+| AFTER_CLASS          |TestItem (type = AFTER_CLASS)  |
+| AFTER_TEST           |TestItem (type = AFTER_TEST)   |
+| AFTER_SUITE          |TestItem (type = AFTER_SUITE)  |
+| AFTER_GROUPS         |TestItem (type = AFTER_GROUPS) |
+
+TestItem – report portal specified object for representing:  suite, test, method objects in different test systems. Used as tree structure and can be recursively placed inside himself.
```
Comment (ksuglotova) [inline README.md]: inside itself

File: README.md (hunk 3 above)
Comment (avarabyeu) [inline README.md]: fixed

File: README.md (hunk 2 above)
Comment (avarabyeu) [inline README.md]: Fixed

File: README.md (hunk 4 above)
Comment (avarabyeu) [inline README.md]: fixed

File: README.md (hunk 5 above)
Comment (avarabyeu) [inline README.md]: fixed

File: README.md (hunk 6 above)
Comment (avarabyeu) [inline README.md]: fixed

**Pull request merged**